pkg/lib/cockpit-po-plugin.js

```diff
@@ -71,6 +68,9 @@ function buildFile(po_file, webpack_module, webpack_compilation) {
                 if (!references.some(str => str.startsWith(`pkg/${config.subdir}`) || str.startsWith('src')))
                     continue;
 
+                if (references.every(filter))
```

Comment:
For the record, I also find this extremely confusing.  This should rather be a predicate which takes the translations which it matches.  ie:  please rewrite that as `if (!references.some(filter)) continue;` and adjust accordingly.